pkg/streamer/streamer.go

```diff
@@ -32,12 +32,25 @@ type Streamer struct {
 type Config struct {
 	Parsers              []parser.PublicationParser // Parsers used to open a publication, in addition to the default parsers.
 	IgnoreDefaultParsers bool                       // When true, only parsers provided in parsers will be used.
-	InferA11yMetadata    bool                       // When true, additional accessibility metadata will be infered from the manifest.
+	InferA11yMetadata    InferA11yMetadata          // When set, additional accessibility metadata will be infered from the manifest.
```

Comment:
I'd say "not empty" instead of "set", since you can't "unset" this by e.g. nulling it. But it's good enough either way